src/sas/sascalc/fit/models.py

```diff
@@ -34,7 +33,7 @@ def plugin_log(message):
     """
     now = time.time()
     stamp = datetime.datetime.fromtimestamp(now).strftime('%Y-%m-%d %H:%M:%S')
-    with io.open(PLUGIN_LOG, 'a', encoding='utf-8') as out:
+    with open(PLUGIN_LOG, 'a', encoding='utf-8') as out:
```

Comment:
I believe this change should be fine at this point, but ant to be sure. @pkienzle, can this be changed to a bare `open` command?